### DISCUSSION TITLE
GitHub Actions Logs in Avast Blacklist

### DISCUSSION THREAD

**Discussion by riquedev** [community/community#122314] [Actions]
### Select Topic Area

Product Feedback

### Body

I'm having a problem (it literally started now) in which Avast notifies the URL that loads the Workflow logs as a threat (marked on the black list).

![image](https://github.com/community/community/assets/18220248/3f4e3e8b-4b5b-4a87-8695-29b438797240)
![image](https://github.com/community/community/assets/18220248/8c7bec3c-84bf-4caa-9a00-7e72b5d2b07e)

**Comment by echoix** [comment]
Having the same thing since yesterday...

**Comment by srbingham** [comment]
Having this issue as well

**Comment by smoravcik** [comment]
same here:

```
https://productionresultssa1.blob.core.windows.net/ - blacklisted
https://productionresultssa2.blob.core.windows.net/ - blacklisted
https://productionresultssa3.blob.core.windows.net/ - working
https://productionresultssa4.blob.core.windows.net/ - blacklisted
https://productionresultssa5.blob.core.windows.net/ - working
https://productionresultssa6.blob.core.windows.net/ - blacklisted
https://productionresultssa7.blob.core.windows.net/ - blacklisted
https://productionresultssa8.blob.core.windows.net/ - working
https://productionresultssa9.blob.core.windows.net/ - working
https://productionresultssa10.blob.core.windows.net/ - working
https://productionresultssa11.blob.core.windows.net/ - working
https://productionresultssa12.blob.core.windows.net/ - working
https://productionresultssa13.blob.core.windows.net/ - blacklisted
https://productionresultssa14.blob.core.windows.net/ - working
https://productionresultssa15.blob.core.windows.net/ - working
https://productionresultssa16.blob.core.windows.net/ - working
https://productionresultssa17.blob.core.windows.net/ - working
https://productionresultssa18.blob.core.windows.net/ - working
https://productionresultssa19.blob.core.windows.net/ - working
```

**Reply by echoix** [reply]
I have the 1, 12, 13, 15, 2, 7, 8 that have been blacklisted and needed to add an exception to see the results of a job.

**Comment by smoravcik** [comment]
Got an update from Avast support. They confirmed false positives and websites are confirmed to be safe. They also stated the whitelist now, however i got 3 and 9 blacklisted again, so I update the ticket with Avast and waiting for full range whitelist.

**Reply by daball** [reply]
This is still happening. I don't even know why I trust Avast anymore.
![image](https://github.com/community/community/assets/703034/5d8be86d-6f0c-4a31-8e3e-f31a2eda9cff)

**Comment by njasm** [comment]
This issue still manifests itself with the latest AVAST antivirus

**Reply by marty0678** [reply]
Yep just found this thread as I just got it too.

**Comment by daball** [comment]
The workaround for me was to uninstall Avast and go on about the rest of my
life. Windows Defender hasn’t been in my way.

**Comment by adielashrov** [comment]
Hey,
Did anyone encounter an issue with Avast, the GitHub co-pilot fails to initiate the GitHub login process?

**Comment by tony-mobilus** [comment]
Simply add an EXCEPTION in your AVAST application.

Menu -> Settings -> Exceptions -> https://productionresultssa*.blob.core.windows.net/*

**Comment by MikeMcC399** [comment]
I'm getting `URL:Blacklist` from Norton 360 for `productionresultssa5.blob.core.windows.net` "because it is a dangerous webpage". First time seeing this.

**Comment by LukasMod** [comment]
In my case this helped:
System Settings → Network → (Wi-Fi / Ethernet) → Details → DNS
Remove these if exist:
```
2a02:a302:0:1::10
2a02:a302:0:1::100
```
